frontend/src/views/ComplianceReports/components/__tests__/ImportantInfoCard.test.jsx

```diff
@@ -0,0 +1,161 @@
+// ImportantInfoCard.test.jsx
+
+import React from 'react'
+import { render, screen, fireEvent, waitFor } from '@testing-library/react'
+import { vi, describe, it, expect, beforeEach } from 'vitest'
+import { ImportantInfoCard } from '../ImportantInfoCard'
+import { useCreateSupplementalReport } from '@/hooks/useComplianceReports'
+import { useNavigate } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import { ThemeProvider } from '@mui/material/styles'
+import theme from '@/themes' // Adjust this path based on your project structure
+
+// Mock the necessary modules
+vi.mock('@/hooks/useComplianceReports')
+vi.mock('react-router-dom', () => ({
+  ...vi.importActual('react-router-dom'),
+  useNavigate: vi.fn()
+}))
+vi.mock('react-i18next', () => ({
+  useTranslation: vi.fn()
+}))
+
+describe('ImportantInfoCard', () => {
+  const complianceReportId = '123'
+  const alertRef = { current: { triggerAlert: vi.fn() } }
+  const navigate = vi.fn()
+  const createSupplementalReportMock = vi.fn()
+
+  beforeEach(() => {
+    vi.resetAllMocks()
+
+    // Mock useNavigate
+    useNavigate.mockReturnValue(navigate)
+
+    // Mock useTranslation
+    useTranslation.mockReturnValue({
+      t: (key) => key
+    })
+
+    // Mock useCreateSupplementalReport
+    useCreateSupplementalReport.mockReturnValue({
+      mutate: createSupplementalReportMock,
+      isLoading: false
+    })
+  })
+
+  // Custom render function to include ThemeProvider
+  const customRender = (ui) => {
```

Comment:
suggestion: Use the standard test wrapper instead of making a new custom render

Reply:
thank you, updated.